fl4health/clients/basic_client.py

```diff
@@ -557,7 +557,7 @@ def setup_client(self, config: Config) -> None:
 
         self.wandb_reporter = ClientWandBReporter.from_config(self.client_name, config)
 
-        self.intialized = True
+        self.initialized = True
```

Comment:
This was the bug

Reply:
Good catch. That is definitely a problem. I guess we don't have a test covering this. As part of this. Do you mind added a test that confirms that this is properly set after `setup_client.py` is run?